js/src/components/SettingsReplacementVariableEditors.js

```diff
@@ -22,13 +24,28 @@ class SettingsReplacementVariableEditors extends React.Component {
 		super( props );
 	}
 
+	filterEditorSpecificReplaceVars( replacementVariables, pageType ) {
+		const {
+			editor_specific_replace_vars: editorSpecificReplaceVars = {},
+			shared_replace_vars: sharedReplaceVars,
+		} = wpseoReplaceVarsL10n;
+
+		const pageTypeSpecificReplaceVars = editorSpecificReplaceVars[ pageType ] || [];
+		const replaceVarNames = [ ...sharedReplaceVars, ...pageTypeSpecificReplaceVars ];
+
+		return replacementVariables.filter( replaceVar => {
+			return replaceVarNames.includes( replaceVar.name );
```

Comment:
I guess this will error in IE11, as far as I know `includes`  it's not supported. For instance, Gutenberg uses lodash `includes()`.